devtools/dump_devinfo.py

```diff
@@ -879,35 +886,32 @@ async def get_smart_fixtures(
         finally:
             await protocol.close()
 
-    device_requests: dict[str, dict] = {}
+    device_requests: dict[str, list[SmartCall]] = {}
     for success in successes:
-        device_request = device_requests.setdefault(success.child_device_id, {})
-        device_request.update(success.request)
+        device_request = device_requests.setdefault(success.child_device_id, [])
+        device_request.append(success)
 
     scrubbed_device_ids = {
         device_id: f"SCRUBBED_CHILD_DEVICE_ID_{index}"
         for index, device_id in enumerate(device_requests.keys())
         if device_id != ""
     }
 
-    final = await _make_requests_or_exit(
-        protocol,
-        device_requests[""],
-        "all successes at once",
-        batch_size,
-        child_device_id="",
+    final = await _make_final_calls(
+        protocol, device_requests[""], "All successes", batch_size, child_device_id=""
```

Comment:
Having a key `""` in the `device_requests` is rather odd, no?

Reply:
Yes but it's always been like that and it just denotes the parent as opposed to the children which have child ids